### PR TITLE
revise environment creation on clean ubuntu 20.04

### PR DESCRIPTION
**Pull request by TokenHung**
Modify env yaml
Adding patch files
Adding pretrained datasets

### CODE REVIEW

File: run_experiments_short.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        # gpcc.run_dataset('8i_longdress_25', 'experiments')
+        # gpcc.run_dataset('8i_loot_25', 'experiments')
+        # gpcc.run_dataset('8i_soldier_25', 'experiments')
+        # gpcc.run_dataset('8i_redandblack_25', 'experiments')
+        # gpcc.run_dataset('8i_longdress_geo_25', 'experiments')
+        # gpcc.run_dataset('8i_loot_geo_25', 'experiments')
+        # gpcc.run_dataset('8i_soldier_geo_25', 'experiments')
+        # gpcc.run_dataset('8i_redandblack_geo_25', 'experiments')
+
+    # vpcc = VPCC()
```
Comment (xtorker) [inline run_experiments_short.py]: I would suggest we also add V-PCC back in the short version.

File: setup_env_ds.sh [hunk 2]
```diff
@@ -72,3 +77,6 @@ patch -sp0 < libgdiam-1.0.3.patch
 cd libgdiam-1.0.3
 mkdir build && cd build && cmake .. && make
 cd ../../../..
+patch -u algorithms/PCGCv2/test.py test.patch
```
Comment (xtorker) [inline setup_env_ds.sh]: What is the patch for?

File: setup_env_ds.sh [hunk 3]
```diff
@@ -72,3 +77,6 @@ patch -sp0 < libgdiam-1.0.3.patch
...
+mv checkpoints/ algorithms/PCGCv1/
```
Comment (xtorker) [inline setup_env_ds.sh:82]: Not needed.

File: setup_env_ds.sh [hunk 4]
```diff
@@ -72,3 +77,6 @@ patch -sp0 < libgdiam-1.0.3.patch
...
+mv ckpts/ algorithms/PCGCv2/
```
Comment (xtorker) [inline setup_env_ds.sh]: Not needed.

**Review by xtorker — CHANGES_REQUESTED** [review]
Please address all the comments. ty
We do not commit any pre-trained model or datasets on GitHub.
Take care next time.

File: setup_env_ds.sh (hunk 2 above)
Comment (TokenHung) [reply setup_env_ds.sh]: remove colors from input point cloud, or the program (test.py) cannot be run

File: run_experiments_short.py (hunk 1 above)
Comment (TokenHung) [reply run_experiments_short.py]: Updated

File: setup_env_ds.sh (hunk 2 above)
Comment (TokenHung) [reply setup_env_ds.sh]: Updated

**Pull request merged**